### ISSUE TITLE
关于公式5和代码实现

### ISSUE THREAD

**Issue by zhangchenxu528** [zju3dv/animatable_nerf#20]
非常感谢开源如此出色的工作。我遇到一个疑问，想请教一下：

1) 公式(5）中bw是由norm(Fw+w_smpl)得到的，其中norm(wi) = wi/sum(w)

2) 代码中使用的是:
bw = self.bw_fc(net)
bw = torch.log(smpl_bw + 1e-9) + bw
bw = F.softmax(bw, dim=1)

3) 我化简代码中的结果是bw = norm(Fw*w_smpl)，不知道是不是我计算错了？希望作者可以帮我答疑一下，非常感谢。

**Comment by pengsida** [comment]
你没有推错, 代码实现没有按照公式5实现, 不过也是residual的一种.

`bw = norm(Fw*w_smpl)`中, w_smpl在0到1之间. 而Fw过了e^x, 所以是大于0的数, 从而改变w_smpl的大小, 过了norm以后仍然为0到1.
